wagtail/admin/templates/wagtailadmin/shared/side_panels/includes/status/workflow.html

```diff
@@ -106,13 +125,45 @@
         {% endif %}
 
         {# Scheduled publishing #}
-        {% if draft_revision and draft_revision.approved_go_live_at %}
-            <div class="w-flex w-space-x-3">
-                {% icon name='info-circle' class_name='w-w-4 w-h-4 w-text-info-100 w-shrink-0' %}
-                <div class="w-label-3 w-flex-1">
-                    {% trans 'This will publish at ' %}{{ draft_revision.approved_go_live_at }}
+        {% if has_draft_publishing_schedule %}
+            <div class="w-mt-3 w-p-4 w-bg-info-50 w-rounded w-flex w-space-x-3">
+                {% if scheduled_go_live_at or scheduled_expire_at %}
+                    {% icon name='calendar-check' class_name='w-w-4 w-h-4 w-text-info-100' %}
+                {% else %}
+                    {% icon name='calendar' class_name='w-w-4 w-h-4 w-text-info-100' %}
+                {% endif %}
+                <div class="w-flex w-flex-1 w-items-start w-justify-between">
+                    <div class="w-flex w-flex-col w-flex-1 w-pr-5 w-space-y-1 w-help-text">
+                        {% if scheduled_go_live_at %}
+                            <div><span class="w-text-primary">{% trans 'Go-live:' %}</span> {{ scheduled_go_live_at }}</div>
+                        {% endif %}
+                        {% if scheduled_expire_at %}
+                            <div><span class="w-text-primary">{% trans 'Expiry:' %}</span> {{ scheduled_expire_at }}</div>
+                        {% endif %}
+                        {% if draft_go_live_at or draft_expire_at %}
+                            <div class="w-label-3">{% trans 'Once published:' %}</div>
+                            {% if draft_go_live_at %}
+                                <div><span class="w-text-primary">{% trans 'Go-live:' %}</span> {{ draft_go_live_at }}</div>
+                            {% endif %}
+                            {% if draft_expire_at %}
+                                <div><span class="w-text-primary">{% trans 'Expiry:' %}</span> {{ draft_expire_at }}</div>
+                            {% endif %}
+                        {% endif %}
+                    </div>
+                    {% if not page_perms.page_locked and not in_explorer %}
+                        {% trans 'Edit schedule' as edit_schedule_text %}
+                        {% dialog_toggle class_name='w-bg-transparent w-text-14 w-p-0 w-text-secondary hover:w-text-secondary-600 w-inline-flex w-justify-center w-transition' dialog_id="schedule-publishing-dialog" text=edit_schedule_text %}
+                    {% endif %}
                 </div>
             </div>
+        {% elif not has_live_publishing_schedule %}
```

Comment:
I’ve found one case where this wasn’t rendering when I was expecting it to. Still investigating.